.github/workflows/ci-all-in-one-build.yml

```diff
@@ -11,19 +11,24 @@ jobs:
   all-in-one:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - name: Harden Runner
+      uses: step-security/harden-runner@2e205a28d0e1da00c5f53b161f4067b052c61f34
+      with:
+        egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
+
+    - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
```

Comment:
@jkowall do you know if dependabot is going to understand this syntax?

Reply:
I do not know how dependabot works, so I cannot answer.

Reply:
Per https://github.com/dependabot/dependabot-core/pull/2492 it should still work.